contracts/staking/StakingFactory.sol

```diff
@@ -296,29 +299,22 @@ contract StakingFactory {
 
         emit InstanceRemoved(instance);
     }
-    
+
     /// @dev Verifies a service staking contract instance.
     /// @param instance Service staking proxy instance.
     /// @return True, if verification is successful.
-    function verifyInstance(address instance) public view returns (bool) {
-        // Get proxy instance params
+    function verifyInstance(address instance) external view returns (bool) {
         InstanceParams storage instanceParams = mapInstanceParams[instance];
-        address implementation = instanceParams.implementation;
-
-        // Check that the implementation corresponds to the proxy instance
-        if (implementation == address(0)) {
-            return false;
-        }
```

Comment:
Why was this removed?

Reply:
This check is now useless, it was left there before the struct with implementation / deployer / isEnabled existed. The `isEnabled` is what matters, and if the `implementation` is zero, the `isEnabled` is false by default.